src/EnergyPlus/HeatBalFiniteDiffManager.cc

```diff
@@ -408,6 +411,23 @@ namespace HeatBalFiniteDiffManager {
         InitialInitHeatBalFiniteDiff(state);
     }
 
+    int setSizeMaxProperties(EnergyPlusData &state)
+    {
+        int numArgs;
+        int numAlphas;
+        int numNumerics;
+        int maxTotalProps = 0;
+
+        state.dataInputProcessing->inputProcessor->getObjectDefMaxArgs(state, "MaterialProperty:PhaseChange", numArgs, numAlphas, numNumerics);
+        maxTotalProps = max(maxTotalProps, numNumerics);
+
+        state.dataInputProcessing->inputProcessor->getObjectDefMaxArgs(
+            state, "MaterialProperty:VariableThermalConductivity", numArgs, numAlphas, numNumerics);
+        maxTotalProps = max(maxTotalProps, numNumerics);
+
+        return maxTotalProps;
+    }
```

Comment:
......... in an ideal world, this would have just been modified to read right from the epJSON, but I know that would have been a big lift just to make this object extensible.  So I won't ask for it right now.